sdc/tests/test_series.py

```diff
@@ -79,6 +79,12 @@
     [np.nan, np.inf, np.inf, np.nan, np.nan, np.nan, np.NINF, np.NZERO]
 ]
 
+test_global_stringmethods = [['one', 'one1', '1', ''],
```

Comment:
`test_global_stringmethods` doesn't look like a generic data, at the least name of the variable is not generic as compared with `test_global_input_data_unicode_kind4`. I think we have to have generic data in the globals.